<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScmEditorColors</class>
 <widget class="QWidget" name="ScmEditorColors">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label_7">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Color set:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="colorSet">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Colorset to view/modify</string>
     </property>
     <item>
      <property name="text">
       <string comment="color-sets" extracomment="'Common' here means &quot;frequent/not rare&quot;, not &quot;shared&quot;">Common Colors</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="color-sets" extracomment="see kdebase/workspace/kcontrol/colors/README.i18n">View</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="color-sets" extracomment="see kdebase/workspace/kcontrol/colors/README.i18n">Window</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="color-sets" extracomment="see kdebase/workspace/kcontrol/colors/README.i18n">Button</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="color-sets" extracomment="see kdebase/workspace/kcontrol/colors/README.i18n">Selection</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="color-sets" extracomment="see kdebase/workspace/kcontrol/colors/README.i18n">Tooltip</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="color-sets" extracomment="see kdebase/workspace/kcontrol/colors/README.i18n">Complementary</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QStackedWidget" name="stackColors">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="pageCommonColors">
      <layout class="QGridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="QTableWidget" name="commonColorTable">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>0</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>1</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>View Background</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>View Text</string>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>Window Background</string>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>Window Text</string>
          </property>
         </item>
         <item row="4" column="0">
          <property name="text">
           <string>Button Background</string>
          </property>
         </item>
         <item row="5" column="0">
          <property name="text">
           <string>Button Text</string>
          </property>
         </item>
         <item row="6" column="0">
          <property name="text">
           <string>Selection Background</string>
          </property>
         </item>
         <item row="7" column="0">
          <property name="text">
           <string>Selection Text</string>
          </property>
         </item>
         <item row="8" column="0">
          <property name="text">
           <string>Selection Inactive Text</string>
          </property>
         </item>
         <item row="9" column="0">
          <property name="text">
           <string>Inactive Text</string>
          </property>
         </item>
         <item row="10" column="0">
          <property name="text">
           <string>Active Text</string>
          </property>
         </item>
         <item row="11" column="0">
          <property name="text">
           <string>Link Text</string>
          </property>
         </item>
         <item row="12" column="0">
          <property name="text">
           <string>Visited Text</string>
          </property>
         </item>
         <item row="13" column="0">
          <property name="text">
           <string>Negative Text</string>
          </property>
         </item>
         <item row="14" column="0">
          <property name="text">
           <string>Neutral Text</string>
          </property>
         </item>
         <item row="15" column="0">
          <property name="text">
           <string>Positive Text</string>
          </property>
         </item>
         <item row="16" column="0">
          <property name="text">
           <string>Focus Decoration</string>
          </property>
         </item>
         <item row="17" column="0">
          <property name="text">
           <string>Hover Decoration</string>
          </property>
         </item>
         <item row="18" column="0">
          <property name="text">
           <string>Tooltip Background</string>
          </property>
         </item>
         <item row="19" column="0">
          <property name="text">
           <string>Tooltip Text</string>
          </property>
         </item>
         <item row="20" column="0">
          <property name="text">
           <string>Active Titlebar</string>
          </property>
         </item>
         <item row="21" column="0">
          <property name="text">
           <string>Active Titlebar Text</string>
          </property>
         </item>
         <item row="22" column="0">
          <property name="text">
           <string>Active Titlebar Secondary</string>
          </property>
         </item>
         <item row="23" column="0">
          <property name="text">
           <string>Inactive Titlebar</string>
          </property>
         </item>
         <item row="24" column="0">
          <property name="text">
           <string>Inactive Titlebar Text</string>
          </property>
         </item>
         <item row="25" column="0">
          <property name="text">
           <string>Inactive Titlebar Secondary</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageSetColors">
      <layout class="QGridLayout" name="_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTableWidget" name="colorTable">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <column/>
         <column/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QStackedWidget" name="stackPreview">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageCommonPreview">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="PreviewWidget" name="colorPreview" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>10</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageSetPreview">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="SetPreviewWidget" name="setPreview" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>10</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PreviewWidget</class>
   <extends>QWidget</extends>
   <header>previewwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SetPreviewWidget</class>
   <extends>QWidget</extends>
   <header>setpreviewwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
